### ISSUE TITLE
Every app as a different cursor (Mir 0.31.1)

### ISSUE THREAD

**Issue by pharaone** [AlanGriffiths/egmde#4]
After installing mir 0.31.1 I noticed that every app as its own cursor:
![gj](https://user-images.githubusercontent.com/17162630/38173648-462d9b64-35c2-11e8-9483-66cc83240e51.gif)

**Comment by AlanGriffiths** [comment]
Yes, applications can draw their own cursors. Why is that an issue?

**Comment by pharaone** [comment]
I think you can set a cursor that is set for every app in egmde, not having many different ones

**Comment by pharaone** [comment]
and the cursor of the weston terminal is the normal one but bigger
